ModelTraining/awsAudioTS2Mp3DownloadandConvert.py

```diff
@@ -0,0 +1,197 @@
+# Get Started
```

Comment:
As a general matter, instructions for how to use a script belongs in the README.

Reply:
I added instructions to the readme